README.md

```diff
@@ -215,6 +214,29 @@ class YourClass {
 }
 ```
 
+## Registering Metrics
+
+The best way to register a metric is via a `static final` class variable as is common with loggers.
+
+```java
+static final Counter requests = Counter.build()
+   .name("requests_total").help("Total requests.").labelNames("path").register();
+```
+ 
+Using the default registry with variables that are `static` is ideal since registering a metric with the same name 
+is not allowed and the default registry is also itself static. You can think of registering a metric, more like 
+registering a definition (as in the `TYPE` and `HELP` sections). The actual samples are tracked via `Child` objects.
```

Comment:
I'm a little wary of mentioning Childs in too much detail, these docs would leave you confused as to what happens for a metric with no labels for example (and most metrics have no labels).